### PR TITLE
Add the workbench conda-recipe

### PR DESCRIPTION
**Pull request by marshallmcdonnell**
Did a copy-paste of the framework recipe and adapted separately to create the workbench package. 

From what I am aware of, it will require further work to get this working similar to the framework package:

- [ ] Get the mantid-builder to update the git header in meta.yml automatically
- [ ] Add the build-deploy step to the Jenkins pipeline

### CODE REVIEW

**Comment by yxqd** [comment]
Looks great! Have one question: do we need sphinx to run mantid workbench? Or we only need it when building it?

File: workbench/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    - librdkafka
+    - readline
+    - openssl
+    - qt
+    - qtconsole
+    - qscintilla2
+    - qtawesome
+    - libglu
+    - sphinx
+    - sphinx_bootstrap_theme
```
Comment (marshallmcdonnell) [inline workbench/meta.yaml]: Probably remove the two sphinx packages here.

**Comment by marshallmcdonnell** [comment]
Good call, I think I was (lazily) just adding what I was missing to both. Maybe others that are not needed for run (and vice versa for build).

**Comment by yxqd** [comment]
libglu is a dependency you added using conda-forge. And it seems you are also using the system opengl glu library. This is interesting. Maybe we can remove the build commands to use system libglu?
https://github.com/mantidproject/conda-recipes/blob/b13fb613bdf3c09f0a45f7070cf7f3875d10504b/workbench/build.sh#L7

**Comment by yxqd** [comment]
Maybe it is easier to just remove libglu from dependency list, to be consistent with the `framework` recipe?

**Comment by marshallmcdonnell** [comment]
I did have to add `libglu` using the conda-forge package since the system install one did not pick it up. Didn't investigate heavily though.

I think maybe moving to the conda-forge `libglu` would be the better way than to use the system library. I can try it out on the framework recipe and add it to this PR (or a separate one if you like) while switching this one.

**Comment by yxqd** [comment]
OK. let us merge it for now. BTW, I just tested your script, commenting out libglu, the builds seem fine. Have not tested them, however

**Comment by marshallmcdonnell** [comment]
Thanks! I can test master now.

**Pull request merged**